src/components/app/header/index.tsx

```diff
@@ -345,3 +346,155 @@ const IconX = () => (
+  ]
+
+  function classNames(...classes: any) {
+    return classes.filter(Boolean).join(' ')
+  }
+
+  return (
+    <div>
+      <Popover className="relative">
+        {({open}) => (
+          <>
+            <Popover.Button
+              className={classNames(
+                open
+                  ? 'hover:bg-gray-100 dark:hover:bg-gray-800'
+                  : 'active:bg-gray-200 dark:hover:text-white',
+                'group rounded-md inline-flex items-center text-base font-medium  focus:outline-none focus:ring-2 focus:ring-offset-2 px-4 py-2 hover:bg-gray-100 dark:hover:bg-gray-800 active:bg-gray-200 dark:hover:text-white transition-all ease-in-out duration-200',
+              )}
+            >
+              <span>Browse</span>
+              <ChevronDownIcon
+                className={classNames(
+                  open
+                    ? 'hover:bg-gray-100 dark:hover:bg-gray-800'
+                    : 'active:bg-gray-200 dark:hover:text-white',
+                  'ml-2 h-5 w-5',
+                )}
+                aria-hidden="true"
+              />
+            </Popover.Button>
+            {/* @ts-expect-error */}
+            <Transition
+              show={open}
+              as={Fragment}
+              enter="transition ease-out duration-200"
+              enterFrom="opacity-0 translate-y-1"
+              enterTo="opacity-100 translate-y-0"
+              leave="transition ease-in duration-150"
+              leaveFrom="opacity-100 translate-y-0"
+              leaveTo="opacity-0 translate-y-1"
+            >
+              <Popover.Panel
+                static
+                className="absolute z-20 mt-3 px-2 w-screen max-w-xl sm:px-0 min-w-max"
+              >
+                <div className="rounded-lg shadow-lg ring-1 ring-black ring-opacity-5 overflow-hidden">
+                  <div className="relative grid grid-cols-flyoutmenu bg-white dark:bg-gray-800 px-7 py-6 gap-1">
+                    {browse.map((item) => (
+                      <a
+                        key={item.name}
+                        href={item.href}
+                        onClick={() => track(`clicked topic`, {resource: item.href})}
+                        className="flex items-start rounded-lg transition ease-in-out duration-150 hover:bg-gray-100 dark:hover:bg-gray-900 py-2 px-3"
+                      >
+                        <p className="text-base font-medium text-gray-700 transition ease-in-out duration-150 dark:text-white hover:text-black">
+                          {item.name}
+                        </p>
+                      </a>
+                    ))}
+                    <div className="mr-6 text-base font-medium  transition ease-in-out duration-150 rounded-lg hover:bg-gray-100 dark:hover:bg-gray-900 py-2 px-3">
+                      <a href="/topics" className="text-blue-500 onClick={() => track(`clicked all topics`, {location})}">
```

Comment:
Looks like the `onClick={() => track(clicked all topics, {location})}` function is declared inside `className` here

Reply:
My bad, I have fixed it